test_tooltool.py

```diff
@@ -64,8 +73,36 @@ def setUp(self):
             self.record_list.append(record)
 
 
+class BaseManifestTest(BaseFileRecordTest):
+
+    def setUp(self):
+        BaseFileRecordTest.setUp(self)
+        self.sample_manifest = tooltool.Manifest([self.test_record])
+        self.sample_manifest_file = 'manifest.tt'
+        self.test_dir = 'test-dir'
+        self.startingwd = os.getcwd()
+        if os.path.exists(self.test_dir):
+            shutil.rmtree(self.test_dir)
```

Comment:
This could lead to race conditions, but meh. Who would run unittests in parallel. :)